scripts/sync-project-board.sh

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env bash
+
+# Copyright 2022 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Vendored from kubernetes-sigs/sig-windows-tools/hack/sync-project-board.sh
+#  * commit: 09de175041779f35595dee339c2463b2eb194555
+#  * link: https://github.com/kubernetes-sigs/sig-windows-tools/blob/09de175041779f35595dee339c2463b2eb194555/hack/sync-project-board.sh
+
+
+set -e
+set -u
+set -o pipefail
+
+## DESCRIPTION:
+##
+## This script queries all repos in a given github org and adds the issues 
+## with label 'sig/auth' to a specified project board.
+##
+## REREQS:
+##
+## This script assumes there is a github PAT in the GITHUB_TOKEN env var
+## that was created with the following permissions:
+##   - repo (all)
+##   - read:org
+##   - user (all)
+##   - read:enterprise
+##   - project (all)
```

Comment:
This seems really broad...

Reply:
I've narrowed it down it the following that works:

```bash
- repo (all)
- read (org)
- project (all)
```

`project (all)` is this which we'll need to mutate the project to add new items

![Screenshot 2023-01-06 at 1 06 29 AM](https://user-images.githubusercontent.com/10608509/210968313-6e86ea4c-cef1-4ad7-92e6-837777e5a94f.png)